### DISCUSSION TITLE
hive auth failure - new install

### DISCUSSION THREAD

**Discussion by 73F0SG1R** [Security-Onion-Solutions/securityonion#5928] [Unsupported Versions]
VERSION: 2.3.80

install went cleanly, so-status all green, soc and fleet work with username/password setup during install, but the hive fails

tried adding user via command line: so-thehive-user-add <email>, but that fails to with (regardless of what I add):

Unable to add user to TheHive; user might already exist
{"type":"AuthenticationError","message":"Authentication failure"}

any help would be appreciated

thanks

**Comment by dougburks** [comment]
Have you tried adding a new user with a different username and a simple password with no special characters?

Are you able to duplicate this on another installation (perhaps just in a VM)?

**Reply by 73F0SG1R** [reply]
thank you, i'll try to replicate the error in a vm

**Comment by hacker0ni** [comment]
Try using `so-user list` to find out which users are available. For some reason only the initial analyst account I've created works properly with TheHive. You can try to create another superuser, change the old account's password within TheHive and then logging in. When you use `so-user update` the password doesn't apply to TheHive and that's mainly the reason for auth failures in my experience.

**Comment by JoeBenutzer** [comment]


**Reply by dougburks** [reply]
@JoeBenutzer From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.

**Reply by JoeBenutzer** [reply]
My apologies, I believed it was the same issue and wished to avoid duplication. I will start a new discussion and delete this one.
